modin/pandas/dataframe.py

```diff
@@ -701,6 +701,26 @@ def eval(self, expr, inplace=False, **kwargs):
                 query_compiler=new_query_compiler
             )
 
+    def fillna(
+        self,
+        value=None,
+        method=None,
+        axis=None,
+        inplace=False,
+        limit=None,
+        downcast=None,
+    ):
+        return super(DataFrame, self)._fillna(
+            False,
+            isinstance(value, Series),
+            value,
+            method,
+            axis,
+            inplace,
+            limit,
+            downcast,
+        )
```

Comment:
For things that are not obvious, I think it's better to use the parameter name. Reading this, it's difficult to guess what's `False` or `isinstance(value, Series)` without going to the function definintion. I think being explicit helps readability:
```suggestion
        return super(DataFrame, self)._fillna(
            squeeze_self=False,
            squeeze_value=isinstance(value, Series),
            value=value,
            method=method,
            axis=axis,
            inplace=inplace,
            limit=limit,
            downcast=downcast,
        )
```